django/db/models/fields/__init__.py

```diff
@@ -2072,7 +2081,19 @@ def formfield(self, **kwargs):
         return super(NullBooleanField, self).formfield(**defaults)
 
 
-class PositiveIntegerField(IntegerField):
+class PositiveIntegerRelDbTypeMixin(object):
+
+    def rel_db_type(self, connection):
+        """
+        Return related field database column data type.
```

Comment:
It would be good to explain why this differs from the base implementation, e.g. what is `connection.features.related_fields_match_type`, etc.